setup.py

```diff
@@ -65,7 +75,7 @@ def fetch_requirements(path):
         include_package_data=True,
         package_data=package_data,
         install_requires=install_requires_list,
-        scripts=scripts_list,
+        entry_points=entry_points,
```

Comment:
what's the benefit of this change?

Reply:
It is a recommended approach for cross-platform compatibility:
https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#scripts